### PR TITLE
Support slot filling

### LINKED ISSUE #16: Slot Filling Support
**Is your feature request related to a problem? Please describe.**
This is currently not related to a bug, but is related to the recent addition of Botmock's Slot Filling support that went live in a recent release.

**Describe the solution you'd like**
The solution here is a way to move the intelligent slot filling solution from Botmock and have it properly populate the export methods properly when moving from the platform into Microsoft Bot Framework.

**Describe alternatives you've considered**
Currently, the only way to accomplish this is to work on a Botmock project after it's imported into Microsoft Bot Framework and build on top. There is no way to properly receive slot filling functions through the export script.

**Additional context**
Will post more information and context here if I end up having more.

### PR DESCRIPTION
**Pull request by nonnontrivial**
resolves #16 

## Includes
- [x] slot filling by way of supporting [conditional response templates](https://github.com/microsoft/BotBuilder-Samples/blob/master/experimental/language-generation/docs/lg-file-format.md#conditional-response-templates)
- [x] documentation updates
- [x] use of Botmock SDK to fetch project assets

### CODE REVIEW

File: lib/sdk.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { EventEmitter } from "events";
+
+interface Config {
+  readonly token: string;
+  readonly teamId: string;
+  readonly projectId: string;
+  readonly boardId: string;
+}
+
+export type JSONResponse = { [assetName: string]: any };
```
Comment (obaid) [inline lib/sdk.ts]: Can this SDK wrapper be extract to the a high-level lib? Perhaps make it part of https://github.com/Botmock/botmock-js ?

File: tsconfig.json [hunk 2]
```diff
@@ -5,5 +5,7 @@
     "sourceMap": true,
```
Comment (obaid) [inline tsconfig.json]: Is there a way for us to have a single `tsconfig.json` that we can pull into any Botmock project to keep things consistent?

File: lib/sdk.ts (hunk 1 above)
Comment (nonnontrivial) [reply lib/sdk.ts]: Created https://github.com/Botmock/botmock-js/issues/10

File: tsconfig.json (hunk 2 above)
Comment (nonnontrivial) [reply tsconfig.json]: Hmmm. One (and maybe not the best) way of doing it is to have a script that is run on `install` which writes the contents of some file in a git repo we point it to as `tsconfig.json`.

**Pull request merged**